server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -131,6 +136,14 @@ protected void configure() {
         Multibinder<Version> supportedVersions = Multibinder.newSetBinder(binder(), Version.class);
         supportedVersions.addBinding().toInstance(Version.DRAFT);
         supportedVersions.addBinding().toInstance(Version.RFC8621);
+
+        Multibinder<Capability> supportedCapabilities = Multibinder.newSetBinder(binder(), Capability.class);
+        supportedCapabilities.addBinding().toInstance(DefaultCapabilities.coreCapability(JmapRfc8621Configuration.UPLOAD_LIMIT_30_MB()));
```

Comment:
@ProvideIntoSet + a separated method so that you can use the actual JmapRfc8621Configuration